tests/example_app/example/page2/templates/page_2.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+  <h2>Currently on page 2</h2>
+
+  <!-- relative static ".static" -->
+  <p>{{ static_url_for('.static', filename='app.css') }}</p>
+
+  <!-- specific static "page_2.static" -->
+  <p>{{ static_url_for('page_2.static', filename='app.css') }}</p>
+
+  <!-- specific static "page_1.static" -->
+  <p>{{ static_url_for('page_1.static', filename='app.css') }}</p>
+
+  <link type="text/css" rel="stylesheet"
+        href="{{ static_url_for('.static', filename='app.css') }}">
+
```

Comment:
Can you remove this extra new line?

Reply:
[27947c4 improved spacing consistency in jinja files](https://github.com/nickjj/flask-static-digest/pull/33/commits/27947c47ef46de2dbfc486e75705926ae6729b41)